### PR TITLE
Fixes stasis beds not working with operation computers

### LINKED ISSUE #48724: Newly built stasis beds have no way of being linked to operating computers
## Reproduction

Have a pre-existing operating computer, build a Lifeform Stasis Unit next to it. It won't be linked, there's no way of linking it. But build the stasis unit first and the operating computer second, and it's fine.

The reason:

This is where operating computers look for tables and stasis beds, which is only called when the operating computer is initialized: https://github.com/tgstation/tgstation/blob/b2be5e5dcab6fa3979b3c13cfa672eaec789b34e/code/game/machinery/computer/Operating.dm#L49

This is how operating tables do it, by manually setting the var instead (which, incidentally, why.):
https://github.com/tgstation/tgstation/blob/53024590decf7325d779d55b79b37424d730ea68/code/game/objects/structures/tables_racks.dm#L463

And this is the code for stasis beds, which are not children of operating tables and never link anywhere:

https://github.com/tgstation/tgstation/blob/4057c7b22c85a719cc65852fe54c1b6397fd1596/code/game/machinery/stasis.dm

Which means if the stasis bed is built after the computer, it will fail this:

https://github.com/tgstation/tgstation/blob/e4b6de4097026cea901ee483d8d6b362ef26ebcd/code/modules/surgery/surgery.dm#L75

High priority bug IMO especially if we're getting rid of cloning.

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Fixes #48724 

Stasis beds will now function the same way as optables (instead of being pulled from max of 7 tiles they need to be placed on the tile next to computer).
Stasis beds, optables and op computers look for eachother when built, and disconnect from eachother when destroyed.
When more than one optables or stasis beds are build, only the newest to link gets to have patient's data on screen.
When both stasis bed and optable is built and linked to the computer, optable takes priority in displaying data.
Tested locally multiple times. 

## Why It's Good For The Game

Fixes things

## Changelog 
:cl: Archanial
add: Operation tables and stasis beds now link to operation computer when built
tweak: Stasis beds needs to be placed on the next tile to a computer
fix: Stasis bed will now display patient's data onto the computer 
/:cl:

### CODE REVIEW

**Comment by Neo-0** [comment]
> fix: Stasis bed will now display patient's data onto the computer

as it hasn't been coded before, it's a feature, not a bugfix

**Comment by Archanial** [comment]
> 
> 
> > fix: Stasis bed will now display patient's data onto the computer
> 
> as it hasn't been coded before, it's a feature, not a bugfix

Right now it shows graphs with no data, so this is more a bugfix to me.

File: code/game/machinery/computer/Operating.dm [hunk 1]
```diff
@@ -11,22 +11,27 @@
 	ui_y = 470
 
 	var/mob/living/carbon/human/patient
-	var/obj/structure/table/optable/table
+	var/obj/structure/table/optable/table = null
+	var/obj/machinery/stasis/sbed = null
```
Comment (kriskog) [inline code/game/machinery/computer/Operating.dm]: No need to null these

File: code/game/machinery/stasis.dm [hunk 2]
```diff
@@ -17,7 +17,20 @@
 	var/stasis_can_toggle = 0
 	var/mattress_state = "stasis_on"
 	var/obj/effect/overlay/vis/mattress_on
-	var/obj/machinery/computer/operating/op_computer
+	var/obj/machinery/computer/operating/op_computer = null
```
Comment (kriskog) [inline code/game/machinery/stasis.dm]: Here too

**Review by kriskog — CHANGES_REQUESTED** [review]

**Review by kriskog — APPROVED** [review]

**Pull request merged**